### ISSUE TITLE
empty_count false positive

### ISSUE THREAD

**Issue by keith** [realm/SwiftLint#1217]
```swift
let foo = count == 0
```

**Comment by marcelofabri** [comment]
Duplicate of #827?

**Comment by keith** [comment]
Yes sorry!
